client/signup/steps/user/index.jsx

```diff
@@ -733,6 +736,7 @@ const ConnectedUser = connect(
 			oauth2Client: getCurrentOAuth2Client( state ),
 			suggestedUsername: getSuggestedUsername( state ),
 			wccomFrom: getWccomFrom( state ),
+			wooPasswordless: getWooPasswordless( state ),
```

Comment:
small nit:
Here its `wooPasswordless` and elsewhere its `isWooPasswordless`. May as well be the same

Reply:
addressed! I made it all wooPasswordless instead